src/index.js

```diff
@@ -0,0 +1,158 @@
+import React from "react";
```

Comment:
@Alihaider168 is this file being used anywhere ? If not, let's get rid of it too.